pkg/controller/services.go

```diff
@@ -661,6 +661,9 @@ func (cont *AciController) apicRedirectPol(name string, tenantName string, nodes
 	monPolDn string, enablePbrTracking bool) (apicapi.ApicObject, string) {
 	rp := apicapi.NewVnsSvcRedirectPol(tenantName, name)
 	rp.SetAttr("thresholdDownAction", "deny")
+	if !cont.config.DisableResilientHashing {
+		rp.SetAttr("resilientHashEnabled", "yes")
+	}
```

Comment:
If we always set to "yes" in:
https://github.com/noironetworks/aci-containers/pull/1525/files#diff-0b386e1ef0eec04bf5d5d93f0d5672149bf33dff5aadb7a3c7e9aa01d7934d3aR640

you can avoid the double negation here and say:
if cont.config.DisableResilientHashing {
		rp.SetAttr("resilientHashEnabled", "no")
	}

Reply:
addressed